docs/source/concepts/local-ledger.rst

```diff
@@ -0,0 +1,296 @@
+  if there exists a shared ledger `G` with the following properties:
+
+  * Whenever `Ls` contains a local ledger `L` for party `A`,
+    the vertices of `L` are exactly the vertices of `G` projected to `A` unless the projection is empty.
+    (This takes multiplicities into account, i.e., if the projections of two vertices in `G` are the same,
+    `L` contains two vertices with the same transactions and it is fixed which vertex in `L` belongs to which vertex in `G`.)
+    Moreover, whenever `L` contains an edge between two vertices, `G` contains an edge between the corresponding vertices in `G`.
+
+Order consistency between local ledgers ensures that the order of transactions is *consistent* across several Participant Nodes and between parties.
+Consistency does not mean, however, that everyone observe the same transactions in the same order.
+The :ref:`causality examples <causality-examples>` below illustrate the fine points.
+
+.. _connection-ledger-model:
+
+Connection to the DAML Ledger Model
+***********************************
+
+A DAML ledger provides better guarantees than order consistency, which are discussed now.
+The shared ledger derived from local ledgers is a acylcic directed graph of transactions
+whereas the Ledger Model formalizes ledgers as a list of commits.
+Fortunately, the shared ledger can be understood as such a list of commits:
+sort the vertices topologically and pair each transaction with its submitter.
+
+This allows us to express the DAML ledger guarantees in terms of the Ledger Model.
+Namely, every DAML ledger ensures (subject to the ledger's trust assumption)
+that there exists a shared ledger for all local ledgers
+such that this shared ledger is :ref:`valid <da-model-validity>` in the sense of the Ledger Model.
+(Note that ledger validity does not depend on the particular topological sort that is chosen.)
+In particular, all transactions on the shared ledger are well-authorized and conform to the DAML model.
+And so are all transactions on the :ref:`Transaction Service <transaction-service>`
+as they are projections of the transactions on the shared ledger.
+
+Transferring consistency to the local ledgers and the transaction stream is trickier as consistency depends on the order of the transactions.
+Not all DAML ledgers enforce the same order at all Participant Nodes because this incurs synchronization, which hinders scalability.
+Participant Nodes are therefore allowed to output events in different orders if these events do not causally depend on each other from the party's perspective.
+Causality is implicitly defined by the conditions for a local ledger.
+The examples in the next section will bring these conditions to life.
+
+These reorderings need not happen on any particular DAML ledger, though.
+In a :ref:`fully centralized DAML ledger <fully-centralized-ledger>`,
+there is by definition only one system that contains a physical copy of the entire shared ledger.
```

Comment:
Yes, but that ordering may be entirely coincidental, e.g. dependent on the order in which the storage backend returns database rows.